challenge.md

```diff
@@ -31,19 +31,19 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
-<!-- Answer Starts Here -->
+Absolute. It is exactly where you will find it.<!-- Answer Starts Here -->
 
 <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->Relative.  When someone else views that font, they won't be able to find it if you use an absolute path on your computer.
```

Comment:
No, that's not correct. When we look at a resource on the internet, we find it by its URL, and a URL is an _absolute_ path. How can we tell? Because whether you refer to that URL on your computer or whether I refer to it on mine, it will still direct us both to the same place.